CIP-XXXX/README.md

```diff
@@ -0,0 +1,176 @@
+---
+CIP: ?
+Title: Royalty Datum Metadata
+Authors: Sam Delaney <sam@ikigaitech.org>
+Comments-URI:
```

Comment:
```suggestion
Discussions:
```
This has changed name (and is also a list, if anything added in the future e.g. Cardano Forum threads).